cw-orch-daemon/src/senders/tx.rs

```diff
@@ -35,4 +39,27 @@ pub trait TxSender: QuerySender {
 
         self.commit_tx_any(msgs, memo)
     }
+
+    /// Uploads the `WasmPath` path specifier on chain.
+    /// The resulting code_id can be extracted from the Transaction result using [cw_orch_core::environment::IndexResponse::uploaded_code_id] and returns the resulting code_id
+    fn upload_wasm(
```

Comment:
I don't think this should be in this trait. Having it as a function on 
`impl CosmosSender` should be sufficient. 

An upload is just a specific case for a tx and will very unlikely be used with anything other than `CosmosSender`.

Reply:
Ok, reverting [6b0742c](https://github.com/AbstractSDK/cw-orchestrator/pull/487/commits/6b0742ce0aa5860dda9cb8a40a7e47f466cdceb6) and creating a function to avoid code duplication